src/t_zset.c

```diff
@@ -1535,6 +1535,9 @@ long zsetRank(robj *zobj, sds ele, int reverse) {
         }
 
         if (eptr != NULL) {
+            if (output_score) {
+                *output_score = zzlGetScore(sptr);
+            }
```

Comment:
nitpick: considering the other `if`s here (below) are without scope braces, think it'll be nicer to drop them for this one line too.
same for the other occurrence below.